pylmod/gradebook.py

```diff
@@ -17,7 +17,19 @@
 
 
 class GradeBook(Base):
-    """API for functions that return gradebook data from MIT LMod service.
+    """
+    All calls to the LMod service return JSON for all calls. The JSON
```

Comment:
Also, it doesn't return the JSON, it returns the python data structure (i.e. a list or dictionary)